### ISSUE TITLE
[Doc] OpenLayers example

### ISSUE THREAD

**Issue by rose-alex42** [visgl/deck.gl#8834]
### Link

https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/openlayers

### Description

The current documentation for OpenLayers results in a Deck with a `controller: false` property and a `style: pointer-events:none;` which removes any possibility of interacting with the Deck MapView. Changing to `controller: true` and `style: pointer-events:auto;` enables Picking to work, but camera controls are broken.

I'm a bit at a loss as there doesn't seem to be any actual working examples anywhere on the net, other than this one, and no documentation on the interoperability between Deckgl and OpenLayers. Reading the code, it feels like there isn't a way for deckgl to tell OpenLayers to change its view. It seems that it's the opposite way, where the OpenLayers Mapview triggers a redraw of Deck when re-rendering.

It feels a bit counter-intuitive to the documentation provided about Overlaid basemaps. From my understanding, it should be Deck telling OpenLayers to redraw when its view changes. Am I completely off track?

Thank you for taking the time to address this.

**Comment by Pessimistress** [comment]
You are correct that in this example, OpenLayers is handling user input and triggering Deck redraw.

In the context of Mapbox/Maplibre/Google Maps, the [overlaid](https://deck.gl/docs/get-started/using-with-map#overlaid) mode all has the base map control deck. The other way around is referred to as [reverse-controlled](https://deck.gl/docs/developer-guide/base-maps/using-with-mapbox#reverse-controlled) in the docs.

Since we only have one example for OpenLayers, it is indeed arbitrary to implement the overlaid approach. It does not mean that you cannot integrate deck with OpenLayers interleaved or reverse-controlled. There is no documentation about them because no one has spent enough time looking into the capability of OpenLayers. I'm happy to help you with such implementations if you are interested in contributing to the documentation.
